src/lib/data-helpers.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Find the next level down of categories
+ * @param {*} taxonomies
+ * @param {*} parent
+ * @returns
+ */
+export const subcategoriesOf = (taxonomies, parent) => {
+  if (taxonomies.length === 0) return []
+  const subcategories = taxonomies.filter(taxon => taxon.slug === parent)
+  return subcategories.length === 0
+    ? []
+    : taxonomies.filter(taxon => taxon.slug === parent)[0].children
+}
+
+/**
+ * Export the days of the week for filters
+ */
+export const formatDaysOptions = days =>
```

Comment:
I've just copied this over from the old code - I wasn't sure if I should remove it since in theory we might start serving all filter options from the API at some point. I think right now its more of a guarantee that the array will be ordered correctly if something weird happens! 😅

Reply:
Mmm, what weird things can happen with hard-coded array and a `.map`? 😄 I think this might be why we have a bloated codebase 😂

Reply:
I just dont trust JS and ordering! has a mind of its own!

Reply:
ah wait, it has an order key in there so we should really be pre-processing it regardless of how js behaves. Unless we take order out of there ofc. 

(went wayyy down a rabbit hole on JS ordering thing - times have changed but we're not there quite yet, looks like next few months/year will see that gotcha eliminated!) 


```
{
    "id": 7,
    "label": "Sunday",
    "slug": "Sunday",
    "order": 7
}
```

Reply:
I'm not sure what you mean about weird JS ordering behaviour - aren't JS arrays just objects under the hood, with each item having a fixed index? And calling `Array.map` preserves the indexing order. What gotchas are you referring to?